docs/src/examples/components/Button/index.tsx

```diff
@@ -16,6 +17,7 @@ const ButtonExamples = () => (
     <States />
     <Variations />
     <Groups />
+    <SplitButton />
```

Comment:
Move out it to separate directory, i.e. `components/SplitButton` as it's a top level component.

Reply:
addressed